Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Builder
+FROM node:alpine as builder
+
+COPY . /src
+
+WORKDIR /src
+
+RUN apk add --no-cache git \
+ && npm install \
+ && npm run build
```

Comment:
this could make smarter use of the cache and intermediate containers.

E.g
```
...
RUN apk add --no-cache git

WORKDIR /src
COPY package.json /src/package.json
RUN npm install

COPY . /src
RUN npm run build
...
```

That way a CSS change will not result in all the NPM dependencies needing to be reinstalled. Unless that is desired ofc

Reply:
Very true. Fixed :)